downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -48,23 +48,22 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 11 (2201.11.0)
 
 ## Ballerina library updates
 
-### Features
+### New features
 
 #### `http` package
 
-- Added static code rules 
-- Added relax data binding support for service and client data binding
+- Added relaxed binding support for service and client data binding
 
 ### Improvements
 
 #### `http` package
 
-- Added `anydata` support for `setPayload` methods in the request and response objects
-- Improved `@http:Query` annotation to overwrite the query parameter name in client
-- Improved `@http:Query` annotation to overwrite the query parameter name in service
-- Added header name mapping support in record fields
-- Migrated client and service data binding lang utils usage into `data.jsondata` module utils `toJson` and `parserAsType`
-- Added support for configuring server name to be used in the SSL SNI extension
+- Added `anydata` support for `setPayload` methods in the request and response objects.
+- Improved the `@http:Query` annotation to overwrite query parameter names in clients.
+- Improved the `@http:Query` annotation to overwrite query parameter names in services.
+- Added header name mapping support in record fields.
+- Migrated client and service data binding to use the `toJson` and `parserAsType` functions from the  `ballerina/data.jsondata` module instead of the `fromJsonWithType` function from the `ballerina.lang.value` module, to support flexible data binding.
```

Comment:
Do we really need this though? @TharmiganK Already covered under new features?

Reply:
What's the improvement then?

Reply:
This allows us to utilize the name annotation to specify different name for the fields rather than the one defined in the type. This is useful when creating record types from OpenAPI schemas where we use Ballerina friendly names for record fields without breaking the data-binding.

Additionally, this migration is a requirement for the relaxed data-binding

Reply:
I think we should mention those. Just the switch isn't an improvement, right? Since these functions are for two different usecases.

Reply:
Mentioned via [d67c961](https://github.com/ballerina-platform/ballerina-dev-website/pull/9655/commits/d67c961656993618c440e9a572fdece777e74b0f)